scrapers/vt/bills.py

```diff
@@ -9,6 +9,41 @@
 
 
 class VTBillScraper(Scraper, LXMLMixin):
+
+    # Dictionary matching bill action phrases to classifications
```

Comment:
Is there a location that our final classification list exists? Is it worth linking to that here?

Reply:
yes there is! I will link